mllm/backends/qnn/passes/CustomOpPatterns.cpp

```
@@ -0,0 +1,96 @@
+#include "CustomOpPatterns.hpp"
+#include "mllm/backends/qnn/QNNBackend.hpp"
+#include "mllm/backends/qnn/QNNUtils.hpp"
+#include "mllm/backends/qnn/CustomLayers.hpp"
+#include "mllm/compile/ir/linalg/Op.hpp"
+#include "mllm/core/DataTypes.hpp"
+#include "mllm/utils/Log.hpp"
+#include "QnnTypes.h"
+
+namespace mllm::qnn {
+
+bool QNNDequantizeAddPattern::addNode(const std::string& graphName, const ir::op_ptr_t& op,
```

Clang (14.0.6)

[error] method 'addNode' can be made static

(readability-convert-member-functions-to-static,-warnings-as-errors)